cmake/test/download_checkmd5.py

```diff
@@ -87,8 +87,13 @@ def download_md5(uri, dest):
     """
     # Create intermediate directories as necessary, #2970
     dirname = os.path.dirname(dest)
-    if len(dirname) and not os.path.exists(dirname):
-        os.makedirs(dirname)
+    if len(dirname):
+        try:
+            os.makedirs(dirname)
+        except OSError, e:
+            if e.errno != 17:
```

Comment:
I would use `errno.EEXIST` instead of just `17`:

https://docs.python.org/2/library/errno.html#errno.EEXIST